iml-agent/src/action_plugins/action_plugin.rs

```diff
@@ -45,6 +45,8 @@ pub fn create_registry() -> action_plugins::Actions {
         .add_plugin("mount_many", lustre::client::mount_many)
         .add_plugin("unmount", lustre::client::unmount)
         .add_plugin("unmount_many", lustre::client::unmount_many)
+        .add_plugin("ha_resource_start", high_availability::start_resource)
+        .add_plugin("ha_resource_stop", high_availability::stop_resource)
```

Comment:
```suggestion
        .add_plugin("ha_start_resource", high_availability::start_resource)
        .add_plugin("ha_stop_resource", high_availability::stop_resource)
```
Just for consistence :)